testsuite/run_sets/secondary_parallelizable.yml

```diff
@@ -73,4 +73,8 @@
 - features/secondary/trad_sp_migration.feature
 - features/secondary/trad_check_registration.feature
 
+# NOT IDEMPOTENT
+
+- features/secondary/minssh_sp_migration.feature
```

Comment:
It will not work here.
You need to add it to the finishing yml, unless we want to create another extra stage for this case.